### PR TITLE
add initial GGA blog post

### PR DESCRIPTION
**Pull request by bgruening**
As promised @erasche ... the initial draft of a GGA blog post announcing annotation.usegalaxy.eu. Its a converted version of your talk.

### CODE REVIEW

**Comment by bgruening** [comment]
@erasche ready when you are :)
We might want to change the date before merging.

File: _posts/2019-08-05-gga.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+location: Freiburg, Germany
+supporters:
+  - galaxy-europe
+  - elixir
+  - denbi
+  - ifb
+---
+
+
+The [Galaxy Genome Annotation (GGA)](https://galaxy-genome-annotation.github.io/) Project is focused on supporting genome annotation inside Galaxy. It consists of several projects and tools that are working closely together to deliver a comprehensive, scalable and easy to use Genome Annotation experience.
```
Comment (erasche) [inline _posts/2019-08-05-gga.md]:
```suggestion
The [Galaxy Genome Annotation (GGA)](https://galaxy-genome-annotation.github.io/) Project is focused on supporting genome annotation inside Galaxy. It consists of several teams, projects, and tool suites that are working closely together to deliver a comprehensive, scalable and easy to use Genome Annotation experience.
```

File: _posts/2019-08-05-gga.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+This blog post will highlight a few of the recent developments and the new Genome Annotation subsection of the European Galaxy server: https://annotation.usegalaxy.eu
+
+![GGA logo](/assets/media/gga-clean.png)
+
+This project was recently presented by Helena Rasche at [ISMB/ECCB 2019](https://docs.google.com/presentation/d/1hJyI1sbfxAzzgoJ5E4eHNicSbYzBDaVwTwfOlOLiV3c).
+
+
+## Collaborative Annotation
+
+[Apollo](https://genomearchitect.readthedocs.io/en/latest/) is a web service for Collaborative Annotation - a “Google Docs of Genome Annotation” if you like to think in Google Terms. It allows you Real-time, collaborative genome annotation, editing, review of your favorite genome.
```
Comment (erasche) [inline _posts/2019-08-05-gga.md]:
```suggestion
[Apollo](https://genomearchitect.readthedocs.io/en/latest/) is a web service for Collaborative Annotation - a “Google Docs of Genome Annotation” if you like to think in Google Terms. It allows for Real-time, collaborative genome annotation, editing, and review of your favorite genome.
```

File: _posts/2019-08-05-gga.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+Since Genome Annotation requires manual curation and review many hands are needed -
```
Comment (erasche) [inline _posts/2019-08-05-gga.md]:
```suggestion
Genome Annotation requires manual curation and review many hands are needed -
```

File: _posts/2019-08-05-gga.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+Biology is a team sport! And Apollo is a wonderful tool to support your team.
+
+![Galaxy Apollo integration](/assets/media/collaborative_editing.png)
+
+To make your life easy we worked closely with the Apollo team and bridged Galaxy with Apollo.
+All your automatic annotations you can do with Galaxy and powerful workflows and afterwards send
```
Comment (erasche) [inline _posts/2019-08-05-gga.md]:
```suggestion
You can do automatic annotation with Galaxy, using its powerful workflow system, and afterwards send
```

File: _posts/2019-08-05-gga.md [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+them to Apollo for manual curation and collaborative editing! A typical Galaxy-Apollo use-case might look like that:
```
Comment (erasche) [inline _posts/2019-08-05-gga.md]:
```suggestion
the data to Apollo for manual curation and collaborative editing! A typical Galaxy-Apollo use-case might look like that:
```

File: _posts/2019-08-05-gga.md [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+  3. Send to Apollo
+  4. Collaboratively Annotate
+  → repeat
+
+![Galaxy Apollo integration](/assets/media/galaxy_apollo.png)
+
+
+## Tools 
+
+The European Galaxy server for Genome Annotation contains a bunch of high-profile tools for structural and functional annotation.
```
Comment (erasche) [inline _posts/2019-08-05-gga.md]:
```suggestion
The European Galaxy server for Genome Annotation contains a wide array of high-profile tools for structural and functional annotation.
```

File: _posts/2019-08-05-gga.md [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+* Automatic annotation
+  * MAKER
+  * Prokka
+
+### Circos
+
+[Circos](http://circos.ca) is a software package for visualizing data and information.
+It visualizes data in a circular layout — this makes Circos ideal for exploring relationships between objects or positions.
+There are other reasons why a circular layout is advantageous, not the least being the fact that it is attractive.
+
```
Comment (erasche) [inline _posts/2019-08-05-gga.md]:
```suggestion
The Circos tool is in final polishing, we will integrate it into training and annotation workflows over the next two months.

```

File: _posts/2019-08-05-gga.md [hunk 8]
```diff
@@ -0,0 +1,140 @@
...
+* SUPERFAMILY: database of structural and functional annotation
+* Coils: Prediction of Coiled Coil Regions in Proteins
+* Gene3d: Structural assignment for whole genes and genomes using the CATH domain structure database
+* SignalP Gram Positive Bacteria
+* SignalP Gram Negative Bacteria
+* SignalP Eukaryotic Bacteria
+* Phobius: combined transmembrane topology and signal peptide predictor
+* TMHMM: Prediction of transmembrane helices in proteins
+
+
```
Comment (erasche) [inline _posts/2019-08-05-gga.md]:
```suggestion
[Try InterProScan in Galaxy!](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/bgruening/interproscan5/interproscan/5.0.0){:target="top"}

```

File: _posts/2019-08-05-gga.md [hunk 9]
```diff
@@ -0,0 +1,140 @@
...
+### MAKER
+
+[MAKER](http://www.yandell-lab.org/software/maker.html) is a portable and easily configurable genome annotation pipeline. Its purpose is to allow smaller eukaryotic and prokaryotic genome projects to independently annotate their genomes and to create genome databases. MAKER identifies repeats, aligns ESTs and proteins to a genome, produces ab-initio gene predictions and automatically synthesizes these data into gene annotations having evidence-based quality values. MAKER is also easily trainable: outputs of preliminary runs can be used to automatically retrain its gene prediction algorithm, producing higher quality gene-models on seusequent runs. MAKER's inputs are minimal and its ouputs can be directly loaded into a GMOD database. They can also be viewed in the Apollo genome browser; this feature of MAKER provides an easy means to annotate, view and edit individual contigs and BACs without the overhead of a database. MAKER should prove especially useful for emerging model organism projects with minimal bioinformatics expertise and computer resources.
+
```
Comment (erasche) [inline _posts/2019-08-05-gga.md]:
```suggestion
[Try MAKER in Galaxy!](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/maker/maker/2.31.10){:target="top"}

```

File: _posts/2019-08-05-gga.md [hunk 10]
```diff
@@ -0,0 +1,140 @@
...
+Functional annotation with a variety of different tools working together and finally handing of the results to Apollo.
+![functional annotation with Galaxy and Apollo](/assets/media/functional_annotation_to_apollo.png)
+
+The same for structural annotation of your Genome. The results will be send to Apollo to enable collaborative editing!
+![functional annotation with Galaxy and Apollo](/assets/media/structural_annotation_to_apollo.png)
+
+## Training
+
+As usual the Galaxy community is providing [hands-on training material](https://galaxyproject.github.io/archive//topics/genome-annotation/) to get your started quickly. Check out our Genome Annotation trainings at: https://galaxyproject.github.io/archive//topics/genome-annotation/ and learn more about MAKER, BUSCO, SNAP/Augustus, Apollo, Tripal querying + admin, Chado querying, JBrowse and Circos.
```
Comment (erasche) [inline _posts/2019-08-05-gga.md]:
```suggestion
As usual the Galaxy community is providing [hands-on training material](https://training.galaxyproject.org/training-material/topics/genome-annotation/) to get your started quickly. Check out our Genome Annotation trainings at: https://training.galaxyproject.org/training-material/topics/genome-annotation/ and learn more about MAKER, BUSCO, SNAP/Augustus, Apollo, Tripal querying + admin, Chado querying, JBrowse and Circos.
```

File: _posts/2019-08-05-gga.md [hunk 11]
```diff
@@ -0,0 +1,140 @@
...
+
+### Prokka
+
+[Prokka](https://github.com/tseemann/prokka) is a software tool to rapidly annotate bacterial, archaeal and viral genomes,
+and produce output files that require only minor tweaking to submit to GenBank/ENA/DDBJ.
+
```
Comment (erasche) [inline _posts/2019-08-05-gga.md]:
```suggestion
[Try Prokka in Galaxy!](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/crs4/prokka/prokka/1.13){:target="top"}

```

File: _posts/2019-08-05-gga.md [hunk 12]
```diff
@@ -0,0 +1,140 @@
...
+### JBrowse
+
+[JBrowse](https://jbrowse.org/) is a fast, embeddable genome browser built completely with JavaScript and HTML5.
+
+![JBrowse](/assets/media/jbrowse.png)
+
```
Comment (erasche) [inline _posts/2019-08-05-gga.md]:
```suggestion
[Try JBrowse in Galaxy!](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/jbrowse/jbrowse/1.16.5+galaxy3){:target="top"}
```

**Comment by bgruening** [comment]
Cool thanks @erasche.

**Comment by bgruening** [comment]
Freel free to merge if you are happy.

**Comment by erasche** [comment]
I definitely am!

**Pull request merged**